index.html

```diff
@@ -1,39 +1,83 @@
-            <p class="book__author">Testeroo Testyy</p>
-          </div>
-          <button class="book__remove-btn">Remove</button>
+  <body>
+    <header class="header">
+      <nav>
+        <p>Awesome Books</p>
+        <ul class="nav-list">
+          <li><a href="#container__list" class="list__nav">List |</a></li>
+          <li><a href="#container__add" class="list__add">Add new |</a></li>
+          <li><a href="#container__contact" class="list__contact">Contact</a></li>
+        </ul>
+      </nav>
+      <p class="date-now"></p>
+    </header>
+    <main>
+      <div class="container__list" id="container__list">
+        <h2 class="books__header--title">All awesome books</h2>
+        <div class="container" id="list">
+          <template class="books__template">
+            <div class="book">
+              <div class="book-description">
+                <p class="book__name">Lorem Ipsum</p>
+                <span>by</span>
+                <p class="book__author">Testeroo Testyy</p>
+              </div>
+              <button class="book__remove-btn">Remove</button>
+            </div>
+          </template>
         </div>
-      </template>
-    </div>
-    <hr class="book__line" />
-    <h2 class="book__addnew">Add a New Book</h2>
-    <form class="add">
-      <input type="text" class="add__title input" id="add__title" name="book_title" placeholder="Title" required />
-      <input type="text" class="add__author input" id="add__author" name="book_author" placeholder="Author" required />
-      <input type="submit" class="add__add-btn input" value="Add" />
-    </form>
-  </main>
-  <script src="app.js"></script>
-</body>
-</html>
\ No newline at end of file
+      </div>
+      <div class="container__add" id="container__add">
+        <h2 class="book__addnew">Add a New Book</h2>
+        <form class="add" id="add">
+          <input
+            type="text"
+            class="add__title input"
+            id="add__title"
+            name="book_title"
+            placeholder="Title"
+            required
+          />
+          <input
+            type="text"
+            class="add__author input"
+            id="add__author"
+            name="book_author"
+            placeholder="Author"
+            required
+          />
+          <input type="submit" class="add__add-btn input" value="Add" />
+        </form>
+      </div>
+      <div class="container__contact" id="container__contact">
+        <h2 class="book__addnew">Contact information</h2>
+        <p class="contact__content">
+          Do you have any questions or do you just want to say "Hello"? You can
+          reach out to us.
+        </p>
+        <ul class="contact__info">
+          <li class="email">Our email</li>
+          <li class="phone">Our phone number</li>
+          <li class="address">Our address</li>
+        </ul>
+      </div>
+    </main>
+    <footer class="footer">
+      <p>Copyright 2021</p>
+    </footer>
```

Comment:
- [ ] Footer has to be to the end of the page but currently when  you open the page it looks like this
![footer](https://user-images.githubusercontent.com/44745855/124966663-da8bbc00-e023-11eb-8674-33d65143b93c.PNG)

Please fix that!